### DISCUSSION TITLE
🚀 Feature Request: 希望loading可以配置默认值

### DISCUSSION THREAD

**Discussion by wekor** [alovajs/alova#542] [Feature Request]
### What problem does the new feature need to solve?

有这样一个需求，我有个列表，我需要加载展示后在展示空或者有数据样子，可是默认为false,就会出现先空在展示加载，在展示有无数据
希望可以配置默认loading

### How important is it to you?

Better to have

**Comment by MeetinaXD** [comment]
请详细描述你是如何使用的，贴上关键代码。

如果使用的是 `useRequest`，那么可以设置 `immediate: true`，在没有配置中间件的情况下，`loading` 会跟随该值默认为 `true`

**Reply by JOU-amjs** [reply]
此问题已修复 @voyageh
